lib/node_modules/@stdlib/lapack/base/dlaswp/README.md

```diff
@@ -0,0 +1,248 @@
+<!--
+
+@license Apache-2.0
+
+Copyright (c) 2024 The Stdlib Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+   http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+
+-->
+
+# dsymv
+
+> Perform a series of row interchanges on the matrix `A` using pivot indices stored in `IPIV`.
```

Comment:
I am not sure if this description is sufficient to explain working of this routine.

Reply:
If anything, for the package level description, you can be more concise.

> Perform a series of row interchanges on an input matrix.